examples/ingenia_control/include/IngeniaProtocol.h

```diff
@@ -0,0 +1,53 @@
+#ifndef KICKCAT_CAN_INGENIA_PROTOCOL_H
+#define KICKCAT_CAN_INGENIA_PROTOCOL_H
+
+namespace kickcat
+{
+    namespace pdo
+    {
+        /// \details See CANOpen DS402 document for description of fields.
+        ///          Here fields are name "Torque" but actually refer to current in amps.
+        struct Input
+        {
+            uint16_t statusWord;             // 0x2011
+            int32_t actualPosition;          // 0x2030 ticks
+            float actualVelocity;            // 0x2031 rev/s
+            float demandTorque;              // 0x2053 Nm
+            float actualTorque;              // 0x2029 Nm
+            float dcVoltage;                 // 0x2060 V
+            int32_t demandPosition;          // 0x2078 ticks
+            int32_t auxiliarPosition;        // 0x2033
+            uint16_t imuAccelX;              // 0x218A ADC steps (to be converted)
```

Comment:
seems to be related to a custom hardware

Reply:
I will remove it then